src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.UnitTests/Infrastructure/BulkUploadFileParserTests.cs

```diff
@@ -133,6 +136,30 @@ public void ProviderRefIsParsedCorrectly()
             Assert.AreEqual("ZB657", result.Last().ProviderRef);
         }
 
+        [Test]
+        public void RecognisePriorLearningIsParsedCorrectly()
+        {
+            var result = _bulkUploadFileParser.GetCsvRecords(_proivderId, _file);
+            Assert.AreEqual(true, result.First().RecognisePriorLearning);
+            Assert.AreEqual(false, result.Last().RecognisePriorLearning);
+        }
```

Comment:
Doesn't the API verify that RPL is only applied to starts after 1-Aug?  

I think I'd rather have all of that logic in one place in the API, rather than rely on clients.  (Theoretically a user could navigate directly to the RPL page on the client and submit it, even though that's extremely unlikely)

If a CSV with a start before 1-Aug *and* RPL is uploaded, then the API should throw it's domain error when creating the draft apprenticeship.

Reply:
But isn't it wrong to default RecognisePriorLearning to false, if the provider hasn't considered it? On the FE the provider has to say yes or no for apprenticeships after 1-Aug

Reply:
In the second line, the provider *has* considered it, and set the entry to `0`, which equates to false.

Reply:
It is now, but should our tests cover accept null values in that column as well as true and false

Reply:
That's what `OptionalFieldsMayBeOmitted` does further down in the file 👍